lib/compilers/v.ts

```diff
@@ -94,7 +94,8 @@ export class VCompiler extends BaseCompiler {
 
     override getDefaultExecOptions(): ExecutionOptions & {env: Record<string, string>} {
         const options = super.getDefaultExecOptions();
-        options.env['VMODULES'] = path.join(path.dirname(this.compiler.exe), '.vmodules');
+        if (process.env.tmpDir === undefined) options.env['VMODULES'] = path.join('/tmp', '.vmodules');
```

Comment:
This is also not possible, because this is not unique and will be shared. It's best to use `this.newTempDir()`, but you'll need to `await` for it, I'm not sure that's possible with this function. I'm afraid you'll have to override `runCompiler()`. Or you could hardcode it to '/app/.vmodules', but that would only work with our specific sandboxing.

Reply:
Ok, it now stores `.vmodules` in the path of the output file